llvm/lib/Target/AIE/AIE2Subtarget.cpp

```diff
@@ -76,3 +76,45 @@ const RegisterBankInfo *AIE2Subtarget::getRegBankInfo() const {
 InstructionSelector *AIE2Subtarget::getInstructionSelector() const {
   return InstSelector.get();
 }
+
+std::bitset<8>
+AIE2Subtarget::getMemoryBanksFromAddressSpace(unsigned AddrSpace) const {
+  std::bitset<8> MemoryBanks;
+
+  switch (AddrSpace) {
+  case AIE2AS::a:
```

Comment:
What about unimplemented cases?  should they return something with all flags set?  or none?

Reply:
Have updated to set as none (assume no conflict)